contribution.md

```diff
@@ -2,37 +2,33 @@
 
 This is an open source project, and we appreciate your help!
 
-Generally, we expect two maintainers to review your pull request before it is approved for merging. Please follow the guidelines specified in below before submitting the pull request.
+Generally, we expect two maintainers to review your pull request before it is approved for merging. Please follow the guidelines below before submitting the pull request.
 
 ## Open Source Licenses
 
-All of the models in the Model Asset Exchange are released under the [Apache License](https://www.apache.org/licenses/LICENSE-2.0) or a similar open-source license. If releasing a pretrained model then the data must also be an open-source license such as the [Community Data License Agreement (CDLA)](https://cdla.io/sharing-1-0/) or [Creative Commons Share-Alike (CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/) licenses.
+All models in the Model Asset Exchange are released under the [Apache License Version 2](https://www.apache.org/licenses/LICENSE-2.0) or a similar open-source license. If releasing a pretrained model then the data must also be released under an open-source license such as the [Community Data License Agreement – Sharing, Version 1.0 (CDLA-Sharing-1.0)](https://cdla.io/sharing-1-0/) (preferred) or [Creative Commons Share-Alike (CC BY-SA 4.0)](https://creativecommons.org/licenses/by-sa/4.0/).
 
 
 ## Merge approval
 
-The project maintainers use LGTM (Looks Good To Me) in comments on the pull request to indicate acceptance prior to merging. A change requires LGTMs from two project maintainers. If the code is written by a maintainer, the change only requires one additional LGTM.
+The project maintainers use LGTM (Looks Good To Me) in comments on the pull request to express approval prior to merging. A change requires LGTMs from two project maintainers. If the pull request is proposed by a maintainer, the change requires only one LGTM.
```

Comment:
This is not how we have it setup. Can we change this to say that it requires approval from 1 maintainer and that maintainer can not be the PR author?